Dockerfile

```diff
@@ -7,23 +7,30 @@ ENV PYTHONUNBUFFERED 1
 WORKDIR /app
 RUN mkdir /entrypoint
 
-RUN apt-get update \
-    && apt-get install -y --no-install-recommends \
-        git \
+COPY --chown=appuser:appuser requirements*.txt /app/
+
+RUN apt-install.sh \
+        build-essential \
         gdal-bin \
-        python3-gdal \
         netcat \
-    && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false \
-    && rm -rf /var/lib/apt/lists/* \
-    && rm -rf /var/cache/apt/archives
-
-COPY --chown=appuser:appuser requirements.txt /app/requirements.txt
-RUN pip install -U pip \
-    && pip install --no-cache-dir -r /app/requirements.txt
+        python3-gdal \
+    && pip install --no-cache-dir \
+        -r /app/requirements.txt \
+        -r /app/requirements-prod.txt \
+    && apt-cleanup.sh \
```

Comment:
@tuomas777 just had issues with running `pip install ...` later in the `Dockerfile` after this `apt-cleanup.sh` has been run. he only managed to make it work by moving `apt-cleanup.sh` to `development` and `production` sections

Reply:
That was fixed by adding `apt-install.sh build-essential` in the `development` stage. See the final revision of that PR: https://github.com/City-of-Helsinki/kukkuu/pull/8/commits/402a09cbe60a76e0ce8b84eab764954e3b57888e

Reply:
yes, we Talked with Tuomas about this. 👍